libraries/stdlib/samples/test/samples/ranges/ranges.kt

```diff
@@ -52,4 +52,34 @@ class Ranges {
         assertTrue(3.14f in range)
         assertFalse(100.1f in range)
     }
+
+    @Sample
+    fun intStep() {
+        val range = 1..6 step 2
```

Comment:
The result is not a range, it's a progression.

Reply:
> The result is not a range, it's a progression.

renamed